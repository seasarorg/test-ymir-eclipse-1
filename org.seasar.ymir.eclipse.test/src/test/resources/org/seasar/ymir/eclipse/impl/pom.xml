<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
         http://maven.apache.org/maven-v4_0_0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>groupId</groupId>
  <artifactId>artifactId</artifactId>
  <version>0.0.1</version>
  <build>
    <plugins>
      <plugin>
        <artifactId>maven-jar-plugin</artifactId>
        <dependencies></dependencies>
      </plugin>
    </plugins>
  </build>
  <repositories>
    <repository>
      <id>www.seasar.org</id>
      <name>The Seasar Foundation Maven2 Repository</name>
      <url>http://maven.seasar.org/maven2</url>
    </repository>
  </repositories>
  <dependencies>
    <dependency>
      <groupId>org.seasar.ymir</groupId>
      <artifactId>ymir-extension</artifactId>
      <version>1.0.0-SNAPSHOT</version>
    </dependency>
    <dependency>
      <groupId>org.seasar.container</groupId>
      <artifactId>s2-extension</artifactId>
      <version>2.4.20</version>
    </dependency>
  </dependencies>
</project>
