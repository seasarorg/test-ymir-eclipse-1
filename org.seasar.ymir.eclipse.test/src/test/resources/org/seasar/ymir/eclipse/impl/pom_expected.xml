<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
         http://maven.apache.org/maven-v4_0_0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>groupId</groupId>
  <artifactId>artifactId</artifactId>
  <version>0.0.1</version>
  <build>
    <plugins>
      <plugin>
        <artifactId>maven-jar-plugin</artifactId>
        <dependencies></dependencies>
      </plugin>
    </plugins>
  </build>
  <repositories>
    <repository>
      <id>www.seasar.org</id>
      <name>The Seasar Foundation Maven2 Repository</name>
      <url>http://maven.seasar.org/maven2</url>
    </repository>
  <repository>
  <id>snapshot.maven.seasar.org</id>
  <name>The Seasar Foundation Maven2 Snapshot Repository</name>
  <url>http://maven.seasar.org/maven2-snapshot</url>
  <snapshots>
    <enabled>true</enabled>
  </snapshots>
</repository>
</repositories>
  <dependencies>
    <dependency>
      <groupId>org.seasar.ymir</groupId>
      <artifactId>ymir-extension</artifactId>
      <version>1.0.0-SNAPSHOT</version>
    </dependency>
    <dependency>
      <groupId>org.seasar.container</groupId>
      <artifactId>s2-extension</artifactId>
      <version>2.4.20</version>
    </dependency>
  <dependency>
  <groupId>group1</groupId>
  <artifactId>artifact1</artifactId>
  <version>version1</version>
</dependency>
</dependencies>
<pluginRepositories>
<pluginRepository>
  <id>codehaus snapshot repository</id>
  <url>http://snapshots.repository.codehaus.org</url>
  <releases>
    <enabled>true</enabled>
  </releases>
</pluginRepository>
</pluginRepositories>
<profiles>
<profile>
  <id>executable</id>
</profile>
</profiles>
</project>
